packages/tokens-studio-for-figma/src/plugin/updateStyles.ts

```diff
@@ -58,7 +58,7 @@ export default async function updateStyles(
     })] : []),
   ]).then((results) => Object.assign({}, ...results));
   if (styleTokens.length < tokens.length && shouldCreate) {
-    notifyUI('Some styles were not created due to your settings. Make sure Ignore first part of token name doesn\'t conflict', { error: true });
+    notifyUI('Some styles were ignored due to Ignore first part of token name setting\'t conflict', { error: true });
```

Comment:
```suggestion
    notifyUI('Some styles were ignored due to "Ignore first part of token name" export setting', { error: true });
```
I know it's [slightly different than suggested](https://github.com/tokens-studio/figma-plugin/issues/2609), but it makes it clearer, no?

Reply:
I think so @cuserox 
@six7 Can you please share your opinion?